auto_round/export/export_to_itrex/export.py

```diff
@@ -145,10 +201,11 @@ def pack_model(
             scale_dtype=scale_dtype,
             zp=zp is not None,
             bias=m.bias is not None,
-            device=device,
-            use_optimum_format=True,
+            device="cuda" if device == "xpu" else device,
```

Comment:
It might be a bit obscure here, can you change it? for example, add a temporary variable quant_device